src/Razor/test/Microsoft.VisualStudio.LanguageServices.Razor.Test/ProjectSystem/FallbackRazorProjectHostTest.cs

```diff
@@ -50,7 +60,8 @@ public void GetChangedAndRemovedDocuments_ReturnsChangedContentAndNoneItems()
                 [ItemReference.FullPathPropertyName] = "C:\\From\\About.cshtml",
             });
             var services = new TestProjectSystemServices("C:\\To\\Test.csproj");
-            var host = new TestFallbackRazorProjectHost(services, Workspace, ProjectManager);
+            var razorProjectChangePublisher = GetRazorProjectChangePublisher();
```

Comment:
All these are fine for now but in the future you could just make a private property and then you'd just need to update the host construction below instead of adding the local `razorProjectChangePublisher ` everywhere`